csv-validator-application/LICENSE

```diff
@@ -0,0 +1,6 @@
+Copyright (c) 2013, The National Archives <digitalpreservation@nationalarchives.gov.uk>
```

Comment:
This file is a duplicate of the one in the project root. It should be removed, the build script should copy in the one from the project root on demand.

Reply:
done